tests/lib/rules-preprocessor/gjs-gts-processor-test.js

```diff
@@ -206,6 +207,37 @@ const invalid = [
       },
     ],
   },
+  {
```

Comment:
is there a way to add the fixed output to this test? is that worth it?

Reply:
You mean the "file content" after auto fix? Not sure if that's worth it. Just need to be sure that the fix range is correct.